docs/rules/no-constructor-return.md

```diff
@@ -0,0 +1,52 @@
+# Disallow returning value in constructor (no-constructor-return)
+
+In JavaScript returning value in constructor is allowed, however this looks meaningless. Forbidding this pattern prevents mistake resulting from unfamiliarity with the language or copy-paste error.
```

Comment:
Re `"however this looks meaningless"`:
To me, it is a bit misleading -- it was not fully meaningless, the return value is the evaluated value when the class is new-ed.

Reply:
How should it be changed?